torchaudio/models/wav2vec2/model.py

```diff
@@ -330,16 +362,55 @@ def wav2vec2_asr_large_lv60k(num_out: int) -> Wav2Vec2Model:
     )
 
 
-def hubert_base() -> Wav2Vec2Model:
+def hubert_base(
+        *,
+        checkpoint: Optional[str] = None,
+        dl_kwargs: Dict[str, Any] = None,
```

Comment:
```suggestion
        dl_kwargs: Optional[Dict[str, Any]] = None,
```

also applies to functions below

Reply:
Good catch. Addressed!